smoke-tests/src/test/resources/templates/jvm.build.gradle.kts

```diff
@@ -24,3 +24,7 @@ dependencies {
     implementation("%DEPENDENCY%")
     testImplementation(kotlin("test"))
 }
+
+kotlin {
+    jvmToolchain(8)
```

Comment:
Shouldn't we add  `org.gradle.toolchains.foojay-resolver-convention` into settings file to resolve toolchains?

Reply:
Nice idea! I've been taken JDK-8 availability for granted.